packages/core/strapi/lib/commands/transfer/export.js

```diff
@@ -104,9 +127,9 @@ module.exports = async (opts) => {
 
     logger.log(`${chalk.bold('Export process has been completed successfully!')}`);
     logger.log(`Export archive is in ${chalk.green(outFile)}`);
-  } catch (e) {
+  } catch {
```

Comment:
We might still want to check if it's a completely unexpected error that wasn't reported by the engine diagnostics, although that's pretty unlikely.